master/pkg/model/task_container_defaults_test.go

```diff
@@ -450,63 +450,19 @@ func TestLogPatternUnmarshal(t *testing.T) {
 		NetworkMode:       "bridge",
 		PreemptionTimeout: DefaultPreemptionTimeout,
 		LogPolicies: expconf.LogPoliciesConfig{
-			expconf.LogPolicy{RawPattern: "test", RawAction: expconf.LogAction{
-				RawExcludeNode: &expconf.LogActionExcludeNode{},
-			}},
-			expconf.LogPolicy{RawPattern: "test2", RawAction: expconf.LogAction{
-				RawCancelRetries: &expconf.LogActionCancelRetries{},
-			}},
+			expconf.LogPolicy{
+				RawPattern: "test",
+				RawActions: expconf.LogActionsV0{expconf.LogActionV0{Type: expconf.LogActionTypeExcludeNode}},
+			},
+			expconf.LogPolicy{
+				RawPattern: "test2",
+				RawActions: expconf.LogActionsV0{expconf.LogActionV0{Type: expconf.LogActionTypeCancelRetries}},
+			},
 		},
 	}
 	require.Equal(t, expected, tcd)
 }
 
-func TestLogPatternPoliciesMerging(t *testing.T) {
```

Comment:
Converted this test to a unit test `log_policy.yaml::legacy_log_policies_merging`